dnf-behave-tests/features/makecache.feature

```diff
@@ -0,0 +1,12 @@
+Feature: makecache command 
+
+
+@not.with_os=rhel__eq__8
+@fixture.httpd
+@bz1745170
+Scenario: disabled makecache --timer does not invalidate cached metadata
+Given I use the http repository based on "dnf-ci-fedora"
+  And I successfully execute dnf with args "makecache"
+ When I execute dnf with args "makecache --timer --setopt=metadata_timer_sync=0"
+  And I execute dnf with args "install setup -v"
+ Then stdout contains "repo: using cache for: http-dnf-ci-fedora"
```

Comment:
@kontura can you please rebase and update this, I think the "http-" prefix is now gone from the name of the repo.